app/controllers/questions_controller.rb

```diff
@@ -1,13 +1,13 @@
 class QuestionsController < ApplicationController
   before_action :set_question, only: [:show, :edit, :update, :destroy]
+  before_action :set_answers, only: [:edit]
```

Comment:
Another fix here. A little complicated to explain, but it was creating multiple blank answers on each update. https://www.sitepoint.com/complex-rails-forms-with-nested-attributes/